runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZonePlugin.java

```diff
@@ -70,6 +74,11 @@
 	// This starts as true since you need to get
 	// above the threshold before sending notifications
 	private boolean absorptionNotificationSend = true;
+	private boolean hitpointsNotificationSend = true;
+	private int lastTickHitpointsVal = 0;
+	private int pointsPerHour = 0;
+	private int gameTime = 0;
```

Comment:
Rename to `nmzSessionTime` or something similar. I thought it was RS's in game clock at first.